yandex scooter tests

ansamoilov opened the pull request:
тесты сервиса яндекс самокат

Commit:
yandex scooter tests

AlexKitIt commented on conftest.py:
```diff
@@ -0,0 +1,22 @@
+@pytest.fixture
+def driver_main(driver):
+    driver.get(URLS['main_page_url'])
+    yield driver
```
Можно лучше: генераторы тут не нужны

AlexKitIt commented on page_object/tests/test_main_page.py:
```diff
@@ -0,0 +1,31 @@
+    @allure.description('Проверяем, что при клике на верхнюю копку "Заказать" открывается форма заказа')
+    def test_open_order_form_via_top_button(self, driver_main):
+        main_page = MainPage(driver_main)
+        main_page.click_to_element(MainPageLocators.ORDER_BUTTON_TOP)
```
Нужно исправить: здесь и далее: методов с принимаемыми локаторами в тестах быть не должно, если это не вариант параметризации, они живут в методах пейджи. В тесте только создание объекта и вызов метода, без указания локаторов элементов с которым происходит взаимодействие. Тест должен быть максимально изолирован от логики выполнения

AlexKitIt commented on page_object/tests/test_main_page.py:
```diff
@@ -0,0 +1,31 @@
+
+    @allure.description('Проверяем, что при клике на вопрос отображается соответствующий ему ответ')
+    @pytest.mark.parametrize('question_number', range(len(ANSWERS)))
+    def test_answer_corresponds_to_question(self, driver_main, question_number):
```
Нужно исправить: здесь и далее: над каждым тестовым методом ставится аннотация allure.title

Commit:
1) убрал генератор из фикстур
2) добавил декораторы allure.title
3) вынес из тестов методы, использующие локаторы, в методы страниц
